### PR TITLE
For bht-002-gclzb replacement the 'preset' key for toZigbee by 'preset_mode' to make it identical with fromZigbee

### PR DESCRIPTION
**Pull request by PeterVoronov**
For bht-002-gclzb replacement the 'preset' key for toZigbee by 'preset_mode' to make it identical with fromZigbee

### CODE REVIEW

File: converters/toZigbee.js [hunk 1]
```diff
@@ -3129,7 +3129,7 @@ const converters = {
         },
     },
     moes_thermostat_mode: {
-        key: ['preset'],
+        key: ['preset_mode'], // to make it identical with fromZigbee
```
Comment (Koenkk) [inline converters/toZigbee.js:3132]: please change this to `key: ['preset', 'preset_mode'],` to prevent a breaking change

Comment (PeterVoronov) [reply converters/toZigbee.js]: > please change this to `key: ['preset', 'preset_mode'],` to prevent a breaking change

Make sense. Done

**Comment by Koenkk** [comment]
Thanks!

**Comment by Koenkk** [comment]
Thinking about this again, the preferred way is `preset` not `preset_mode`. So I've reverted this and add `preset`  to thefromZigbee converter.

**Comment by sjorge** [comment]
Should we also do this for the eurotonic trv’s boost mode?

currently we hack around with system_mode, which... is not really great.

**Comment by PeterVoronov** [comment]
> Thinking about this again, the preferred way is `preset` not `preset_mode`. So I've reverted this and add `preset` to the fromZigbee converter.

Looks better 


> Should we also do this for the eurotonic trv’s boost mode?
> 
> currently we hack around with system_mode, which... is not really great.

May be simple - "mode" ? Or status ?
`case tuya.dataPoints.state: // Thermostat on standby = OFF, running = ON
                return {system_mode: value ? 'heat' : 'off', mode: value ? 'heat' : 'off'};`

**Comment by Koenkk** [comment]
`system_mode`  is heavily used already so I don't think we should change it. 

@sjorge 

> Should we also do this for the eurotonic trv’s boost mode?

Don't know the details about this but sounds good

**Pull request merged**